.github/CONTRIBUTING.md

```diff
@@ -117,7 +117,7 @@ Once you create a pull request, the tests will automatically run. It is a good p
 ### Run one test
 
 1. Find the folder containing the e2e test that you're looking for in `cypress/e2e`.
-2. Run `npm test FOLDER` and change FOLDER with the folder from the previous step (example: `run test scoped_elements`).
+2. Run `SINGLE_TEST=FOLDER npm test` and change FOLDER with the folder from the previous step (example: `SINGLE_TEST=scoped_elements run test scoped_elements`).
```

Comment:
good catch, I think it should be `SINGLE_TEST=SINGLE_TEST=streaming npm test npm test` though

Reply:
I think you meant `SINGLE_TEST=scoped_elements run test`

Reply:
yep